module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -1092,12 +1092,14 @@ public function placeHold($holdDetails)
     /**
      * Get FOLIO hold IDs for use in cancelHolds.
      *
-     * @param array $hold An single request
-     * array from getMyHolds
+     * @param array $hold   A single hold array from getMyHolds
+     * @param array $patron Patron information from patronLogin
      *
      * @return string request ID for this request
+     *
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
      */
-    public function getCancelHoldDetails($hold)
+    public function getCancelHoldDetails($hold, $patron)
```

Comment:
I think a default parameter was missed on this one.

Reply:
Oops. Fixed.

Reply:
Thanks!